common/src/main/java/com/wynntils/utils/render/buffered/CustomRenderType.java

```diff
@@ -89,10 +89,30 @@ public class CustomRenderType extends RenderType {
                             .setWriteMaskState(COLOR_WRITE)
                             .createCompositeState(false)));
 
+    private static final Function<ResourceLocation, RenderType> LOOTRUN_QUAD =
+            Util.memoize(resource -> RenderType.create(
+                    "wynntils_lootrun_quad",
+                    DefaultVertexFormat.POSITION_COLOR_TEX,
+                    Mode.QUADS,
+                    256,
+                    false,
+                    false,
+                    CompositeState.builder()
+                            .setShaderState(POSITION_COLOR_TEX_SHADER)
+                            .setCullState(NO_CULL)
+                            .setTextureState(new TextureStateShard(resource, false, false))
+                            .setTransparencyState(CustomRenderStateShard.SEMI_TRANSPARENT_TRANSPARENCY)
+                            .setWriteMaskState(COLOR_WRITE)
+                            .createCompositeState(false)));
```

Comment:
We don't need to use memoize here, do we? We can just hard-code the lootrun texture here and it will be a bit simpler.